src/chapters/angular-lsn2/studio.rst

```diff
@@ -0,0 +1,265 @@
+Studio: Angular, Part 2
```

Comment:
Blake, I think that this is a really strong studio.  Nice work.

The code meshes well with the examples and exercises given in the chapter.